app/controllers/users_controller.rb

```diff
@@ -10,7 +10,7 @@ def show
   end
 
   def index
-    @pagy, @users = pagy(@users,  items: Company::PAGE_SIZE)
+    @pagy, @users = pagy(@users,  items: User::PAGE_SIZE)
```

Comment:
The PAGE_SIZE defined in the company model is the standard for all resources. You don't need to declare a new one. Just use that everywhere

Reply:
done